app/kumactl/cmd/completion/testdata/bash.golden

```diff
@@ -1243,6 +1243,10 @@ _kumactl_get_global-secrets()
     flags_with_completion=()
     flags_completion=()
 
+    flags+=("--offset=")
+    two_word_flags+=("--offset")
+    flags+=("--size=")
+    two_word_flags+=("--size")
```

Comment:
For some reasons there was no pagination on secrets and global secrets. @jakubdyszkiewicz do you know if it was for a specific reason?